.github/workflows/build-emsdk.yml

```diff
@@ -67,13 +68,17 @@ jobs:
     - name: Install python-wasm-sdk
       run: |
         sudo apt-get install lz4
-        mkdir python-wasm-sdk && cd python-wasm-sdk
-        curl -sL --retry 5 https://github.com/pygame-web/python-wasm-sdk/releases/download/$SDK_VERSION/$SDK_ARCHIVE | tar xvP --use-compress-program=lz4
+        echo https://github.com/pygame-web/python-wasm-sdk/releases/download/$SDK_VERSION/$SDK_ARCHIVE
+        #mkdir python-wasm-sdk && cd python-wasm-sdk
+        # ??? curl -sL --retry 5 https://github.com/pygame-web/python-wasm-sdk/releases/download/$SDK_VERSION/$SDK_ARCHIVE | tar xvP --use-compress-program=lz4
+        wget -O- https://github.com/pygame-web/python-wasm-sdk/releases/download/3.1.19.0/python3.11-wasm-sdk-Ubuntu-22.04.tar.lz4 | tar xvP --use-compress-program=lz4
```

Comment:
I don't get why this change is being made. Hardcoding the URL would mean that the previously set enviroment variables are redundant.

I'd prefer if this entire diff is reverted (except for the rm SDL1 part)

Reply:
i saw a string bug in CI execution, and i thought curl line was problematic, when i switched to wget it worked and i just forgot about it. That's why reviewing is so important :)